docs/sdk-and-tools/notifier.md

```diff
@@ -349,12 +363,30 @@ It is recommended to use the setup with RabbitMQ, if it is very important to avo
 
 [comment]: # (mx-context-auto)
 
+### WebSocket
+
+If WebSocket subscribing solution is selected via CLI parameter, an additional http
+endpoint `/hub/ws/` will be available for sending subscriptions.
+
+There are more notes
+on how to send subscriptions and how to consume events
+[here](https://github.com/multiversx/mx-chain-notifier-go?tab=readme-ov-file#websockets).
+
+Please check also events section [below](#events) on how WS event is constructed. 
+
+:::info
+Please make the distinction between `observer node` -> `notifier` communication which can be done via WebSocket and
```

Comment:
_... and the ..._; also, maybe Websocket as `Websocket` in both places.

Reply:
done